Sources/VLCSettingsController.m

```diff
@@ -125,12 +138,38 @@ - (void)updateUIAndCoreSpotlightForPasscodeSetting:(BOOL)passcodeOn
 
 #pragma mark - IASKSettings delegate
 
-- (void)settingsViewController:(IASKAppSettingsViewController*)sender buttonTappedForSpecifier:(IASKSpecifier*)specifier
+- (IBAction)pushAboutViewController:(id)sender
 {
     VLCAboutViewController *aboutVC = [[VLCAboutViewController alloc] init];
     [self.navigationController pushViewController:aboutVC animated:YES];
 }
 
+- (UIView *)settingsViewController:(id<IASKViewController>)settingsViewController tableView:(UITableView *)tableView viewForHeaderForSection:(NSInteger)section
+{
+    UIView *header = [[UIView alloc] initWithFrame:CGRectMake(20, 0, tableView.bounds.size.width, 64)];
```

Comment:
I think we should consider using a macro at the top for the default values.
For example something like:
`#define SETTINGS_HEADER_HEIGHT 64`

We could also use a `const` for this if you prefer.

Reply:
Yes, that's a good idea and matches what we have in other classes.